<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>King's Field Randomizer!</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/res/kraken.png</normaloff>:/res/res/kraken.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0">
    <item>
     <widget class="QFrame" name="filePickFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <item>
        <widget class="QLabel" name="fileLabel">
         <property name="text">
          <string>Your FDAT.T file:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pickFileButton">
         <property name="text">
          <string>click to choose...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="md5Label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>King's Field Randomizer
by IvanDSM

Extract FDAT.T from the CD/COM/ folder in your CD
and replace it with the randomizer's output.
For replacing FDAT.T in a CD image I recommend using CDmage.

I know this isn't the easiest method but it's what I can do for now :(
Reading .bin files turned out to be quite a pain...</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="monsterCheck">
      <property name="text">
       <string>Randomize monsters</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="itemDropCheck">
      <property name="text">
       <string>Randomize monster item drops</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="noEmptyDropsCheck">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>All monsters should drop items</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="itemCheck">
      <property name="text">
       <string>Randomize items (unfinished, BREAKS THE GAME)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="statusView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="randomizeButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Randomize!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
